SPECS/golang/golang.spec

```diff
@@ -14,7 +14,7 @@
 %define __find_requires %{nil}
 Summary:        Go
 Name:           golang
```

Comment:
I believe we only needed to upgrade msft-golang.

Reply:
Sure:
- https://github.com/microsoft/azurelinux/pull/11921

PTAL whenever you get a chance.